### PR TITLE
feat(ui): add reserved ranges table

### PR DESCRIPTION
**Pull request by huwshimi**
## Done

- Add the reserved ranges table to the subnets and vlans details pages.

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- Using the angular client go to a subnet and add some reserved ranges, do the same for a VLAN.
- Visit the subnet and vlan details for the subnet and VLAN you added the reserved ranges to.
- You should see the reserved ranges table with the same data as the angular client.

## Fixes

Fixes: canonical-web-and-design/app-tribe#662.

### CODE REVIEW

File: ui/src/app/subnets/views/SubnetsList/_index.scss [hunk 1]
```diff
@@ -1,56 +1,58 @@
-.subnets-table {
-  tr:not(:first-child) {
-    border-top: 0 !important;
-  }
-  tbody tr:not(:first-child) td {
-    border-top: 1px solid $color-mid-light;
-  }
+@mixin SubnetsList {
```
Comment (huwshimi) [inline ui/src/app/subnets/views/SubnetsList/_index.scss:1]: This was a drive-by fix to include this class in a mixin. Below this line is all indentation changes.

**Review by Caleb-Ellis — APPROVED** [review]
LGTM!
